server/service/src/backend_plugin/boajs/call_plugin.rs

```diff
@@ -30,16 +31,22 @@ impl PartialEq for BoaJsPluginError {
     }
 }
 
+fn plugin_type_to_string(r#type: &PluginType) -> String {
+    serde_json::to_string(r#type).unwrap().replace("\"", "")
+}
+
 pub(crate) fn call_plugin<I, O>(
     input: I,
-    name: &str,
+    r#type: &PluginType,
```

Comment:
I like this change even though it's a little bit out of scope, it avoids having to hard code string and makes PluginType source of truth